app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,104 @@
+class TweetsController < ApplicationController
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/new'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    tweet = Tweet.new(params[:tweet])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.new(content: params[:content], user: current_user)
```